Packages/com.unity.inputsystem/InputSystem/State/InputState.cs

```diff
@@ -90,11 +90,29 @@ public static unsafe void Change(InputDevice device, InputEventPtr eventPtr, Inp
         /// also performs related tasks such as checking state change monitors, flipping buffers, or making the respective
         /// device current.
         /// </remarks>
-        public static void Change<TState>(InputControl control, TState state, InputUpdateType updateType = default,
-            InputEventPtr eventPtr = default)
-            where TState : struct
+        public static void Change<TState>(InputControl control, TState state, InputUpdateType updateType = default, InputEventPtr eventPtr = default)
+                where TState : struct
+        {
+            double time = eventPtr.valid ? default : InputRuntime.s_Instance.currentTime; 
+            Change(control, ref state, updateType, eventPtr, time);
+        }
+
+        /// <summary>
+        /// Perform one update of input state specifying the time
+        /// </summary>
+        /// <remarks>
+        /// Incorporates the given state and triggers all state change monitors as needed.
+        ///
+        /// Note that input state changes performed with this method will not be visible on remotes as they will bypass
+        /// event processing. It is effectively equivalent to directly writing into input state memory except that it
+        /// also performs related tasks such as checking state change monitors, flipping buffers, or making the respective
+        /// device current.
+        /// </remarks>
+        public static unsafe void Change<TState>(InputControl control, TState state, double time, InputUpdateType updateType = default)
+                where TState : struct
```

Comment:
I only added this function as I first add a new default "time" parameter to the existing function but that caused the CI tooling complain that I've changed an API and this is not allowed without a version change.

I was just trying to avoid that restriction by not changing an existing API.

However I've still had to add the time parameter to the "guts" unsafe Change<>() function - it's not clear to me that the unsafe function shouldn't be an internal API... but I guess I can't change that now.

It all seems a bit of mess to me - I just want to add a new benign default parameter that we can optionally use to avoid a redundant call back to native the get the time.